extra/redisotel/config.go

```diff
@@ -63,7 +62,7 @@ func newConfig(opts ...baseOption) *config {
 		opt.apply(conf)
 	}
 
-	conf.attrs = append(conf.attrs, semconv.DBSystemKey.String(conf.dbSystem))
+	conf.attrs = append(conf.attrs, semconv.DBSystemRedis)
```

Comment:
@esara looks like this should be reverted as well.

Reply:
no problem, I was trying for semconv, where possible, but it is not crucial to this pr - reverted